buildout.cfg

```diff
@@ -45,6 +45,7 @@ coverage =
 # baixado de novo: o fato dele estar no src não impede o download
 # dele de novo como um egg se ele estiver pinado no extends.
 brasil.gov.tiles =
+collective.polls = 1.10b1
```

Comment:
Need to remove this after this PR is merged https://github.com/plonegovbr/portal.buildout/pull/85/files